frontend/src/pages/frontPage/frontpage.module.css

```diff
@@ -0,0 +1,182 @@
+.panel {
+    width: 100%;
+    height: 20em;
+}
+
+.panelContainer{
+    position: relative;
+}
+.panelText{
+    position:absolute;
+    top: 5em;
+    right: 2em;
+    left: 20em;
+}
+
+.mainContainer{
+    display: flex;
+    flex-direction: column;
+}
+
+.mainContainer > h3{
+    font-size: 1.5em;
+    padding: 1em;
+    letter-spacing: .1em;
+}
+.coursesContainer{
+    display:flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.courseContainer{
+    display: flex;
+    flex-direction: row;
+    padding: 1em;
+    margin: 1em;
+    border-style: solid;
+    border-color:#122a50 ;
+    border-width: .15em;
```

Comment:
Instead of three different lines you can connect them in one
`border: #122A50 0.15em solid:`